lib/src/shared/presentation/widgets/custom_button_widget.dart

```diff
@@ -12,6 +12,7 @@ class CustomButtonWidget extends StatelessWidget with ButtonStyles {
   final Color? disabledTextColor;
   final bool isButtonDisabled;
   final bool showBorder;
+  final EdgeInsets? padding;
```

Comment:
Why ? This require?

Reply:
to controll the padding of button from out side